ingestion-edge/tests/integration/conftest.py

```diff
@@ -17,11 +21,11 @@ def pytest_addoption(parser: _pytest.config.argparsing.Parser):
         help="Server to run tests against",
     )
     parser.addoption(
-        "--no-create-pubsub-resources",
-        action="store_false",
-        dest="create_pubsub_resources",
-        default=True,
-        help="Don't create PubSub resources for tests",
```

Comment:
this option doesn't really make sense imo after moving pubsub resources to pytest fixtures